storage/pipeline.go

```diff
@@ -150,6 +151,31 @@ func (c *DBClient) GetPipeline(id string) (Pipeline, error) {
 	return pipeline, nil
 }
 
+// GetPipelinesByDay returns all pipelines that have the start-day equal the param.
+func (c *DBClient) GetPipelinesByDay(day int) ([]Pipeline, error) {
+	var pipelines []Pipeline
+
+	collection := c.mgoClient.Database(database).Collection(pipelineCollection)
+	itens, err := collection.Find(context.TODO(), bson.D{{Key: "start-day", Value: day}})
+	if err != nil {
+		if err == mongo.ErrNoDocuments {
+			return []Pipeline{}, nil
+		}
+		return nil, fmt.Errorf("error getting pipelines. Find error: %q", err)
+	}
+
+	for itens.Next(context.Background()) {
```

Comment:
itens também tem um método `All`, que preenche a lista direto (basicamente o que você está fazendo).

Exemplo [aqui](https://www.mongodb.com/blog/post/quick-start-golang--mongodb--how-to-read-documents)

Reply:
Ah sim, maravilha. Vou adicionar. E fazer o merge na outra branch e atualizar o outro pr pra você poder revisar.